schemas/geologic_formation.py

```diff
@@ -5,6 +5,19 @@
 from schemas import BaseResponseModel
 
 
+# ------ CREATE ----------
+class CreateGeologicFormation(BaseModel):
+    """
+    Schema for creating a geologic formation.
+    Used during data transfer and API creation.
+    """
+
+    formation_code: str | None = None
+    description: str | None = None
+    lithology: str | None = None
```

Comment:
because this is a restricted vocabulary you can use the enum created from the lexicon here